web/pdf_find_bar.js

```diff
@@ -33,6 +33,7 @@ class PDFFindBar {
     this.highlightAll = options.highlightAllCheckbox;
     this.caseSensitive = options.caseSensitiveCheckbox;
     this.entireWord = options.entireWordCheckbox;
+    this.matchDiacritics = options.matchDiacriticsCheckbox;
```

Comment:
Nit: Let's move this up one line, since the new checkbox is placed before the entireWord-one in the findbar (similar with the addEventListener placement below).